astropy/cosmology/core.py

```diff
@@ -76,6 +76,10 @@ class Cosmology(metaclass=abc.ABCMeta):
 
     For details on how to create performant custom subclasses, see the
     documentation on :ref:`astropy-cosmology-fast-integrals`.
+
+    This class accepts a class keyword-argument ``skip_registration`` (default
```

Comment:
Don't forget to remove this addition to the docstring!  Or, maybe better, replace it with something that states that by default classes are registered, and one can avoid that by defining a no-op `_register_cls` (or do that in the `_register_cls` docstring?)

Reply:
Done!